plugin/upgrade.xml

```diff
@@ -218,4 +218,32 @@
           </migrate-properties>
       </migrate-command>
   </migrate>
+  <migrate to-version="26">
+      <migrate-command name="Create Dynamic Deployment group">
+          <migrate-properties>
+              <migrate-property name="securityDomain" default=""/>
+          </migrate-properties>
+      </migrate-command>
+      <migrate-command name="Create Static Deployment group">
+          <migrate-properties>
+              <migrate-property name="securityDomain" default=""/>
+          </migrate-properties>
+      </migrate-command>
+      <migrate-command name="Deploy Deployment group">
+          <migrate-properties>
+              <migrate-property name="srcsecurityDomain" default=""/>
+              <migrate-property name="securityDomain" default=""/>
```

Comment:
For the future, you can migrate property names, for example
`<migrate-property name="tarsecurityDomain" old="securityDomain" default=""/>`

Reply:
Thanks! But in this case there is only new properties as all Security Domain props where missing.